### PR TITLE
Mark 2.0.20 as shipped

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
@dschuff maybe you want to trigger and LTO build of emscripten-releases DEPS@e0c15cd14170f407a9eb27fcbad22931dc67feb7?

**Review by dschuff — APPROVED** [review]

**Comment by dschuff** [comment]
Yeah I can do that. I'll probably do it after my 3 outstanding infra CLs all land though. I had intended to hold off on all 3 until we had our release cut, but you CQ+2'd one of them and it landed, so I just went and did the others too.

**Review by kripken — APPROVED** [review]

**Pull request merged**
